extensions/blocks/instagram-gallery/store/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { combineReducers, registerStore } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import * as actions from './actions';
+import * as reducers from './reducers';
+import * as selectors from './selectors';
+
+const reducer = combineReducers( reducers );
+
+export default registerStore( 'jetpack/instagram-gallery', {
```

Comment:
Should we pull the name from `index.js`, here and when retrieving data in `use-connect-instagram.js`?

Reply:
I thought of that, but ended up thinking that the added value of doing it would be pretty negligible.
I think we'd still need to build the string (`'jetpack/' + name`), and the store name would become not directly searchable in the codebase.

Also, there is only another case of a block-related store, `jetpack/publicize`, and it's hardcoded as well:

https://github.com/Automattic/jetpack/blob/54ccef3595ddc3cd1371b3a7f7ad135f11919f22/extensions/blocks/publicize/store/index.js#L15